Cartfile

```diff
@@ -1 +1 @@
-github "ReactiveX/RxSwift" ~> 4.0
+github "ReactiveX/RxSwift" ~> 4.4.0
```

Comment:
No reason to bump the minimum version. We can just do Carthage update to make sure its resolved to the latest - but you'd probably want people who use older versions of RxSwift to be able to use this without a prebuilt binary

Reply:
Should we keep `4.0` or `4.4`?